src/framework/preferences/js/ArrowScrolling.js

```diff
@@ -118,4 +142,18 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     };
 
+    fluid.prefs.arrowScrolling.getClosestPanelIndex = function (panels) {
+        var panelArray = [];
+        panels.each(function (idx, panel) {
+            panelArray.push({
+                index: idx,
+                offset: Math.abs($(panel).offset().left)
+            });
+        });
+        panelArray.sort(function (a, b) {
+            return a.offset - b.offset;
+        });
+        return panelArray[0].index;
```

Comment:
This is the first point we might get a failure when there are 0 panels - probably some kind of guarding is appropriate